cpp/dolfinx/fem/assembler.cpp

```diff
@@ -186,19 +229,10 @@ void fem::assemble_matrix(
 void fem::assemble_matrix(Mat A, const Form& a, const std::vector<bool>& bc0,
                           const std::vector<bool>& bc1)
 {
-  const std::function<int(PetscInt, const PetscInt*, PetscInt, const PetscInt*,
-                          const PetscScalar*)>
-      mat_set_values_local
-      = [&A](PetscInt nrow, const PetscInt* rows, PetscInt ncol,
-             const PetscInt* cols, const PetscScalar* y) {
-          PetscErrorCode ierr
-              = MatSetValuesLocal(A, nrow, rows, ncol, cols, y, ADD_VALUES);
-#ifdef DEBUG
-          if (ierr != 0)
-            la::petsc_error(ierr, __FILE__, "MatSetValuesLocal");
-#endif
-          return 0;
-        };
+  std::vector<PetscInt> sp;
```

Comment:
`sp` is a bit cryptic.

Reply:
I don't know why I called it that, in fact.